themes/classic/_dev/js/components/product-select.js

```diff
@@ -32,6 +32,7 @@ export default class ProductSelect {
     let $arrows =   $('.js-modal-arrows');
     let $thumbnails = $('.js-modal-product-images');
     let $onsale =  $('.on-sale');
+    let $new = $('.new');
```

Comment:
Why create this $new variable if you're not using it line 53 :thinking:

Reply:
Do the same with `$('.pack')`

Reply:
I forgot to remove it.  I add a new variable for pack.